### ISSUE TITLE
 NativeMessage, got unknown command.

### ISSUE THREAD

**Issue by aimuz** [bitwarden/clients#11220]
### Steps To Reproduce

The Bitwarden plugin for browsers doesn't biometric unlock properly, my bitwarden desktop is launched, it's a sudden behavior, I tried restarting and still have the problem. But bitwarden desktop can fingerprint unlock properly. Only biometric unlocking via the browser's Bitwarden extension has the problem

### Expected Result

Able to unlock with normal fingerprints

### Actual Result

Unable to unlock.

I got the following logs that might be useful

```text
received event of :  message {
  appId: '35483e36-4468-4187-8d07-1d31eb919204',
  message: {
    encryptedString: 'xxx',
    encryptionType: 2,
    data: 'xxx',
    iv: 'xxx',
    mac: 'xxx'
  }
}
23:01:32.810 › NativeMessage, got unknown command.
```

All that's certain is that there's no problem with communications.

When I turn off the extension's biometrics and turn it back on again the first verification works fine, but then I immediately lock the extension's password vault and then I can't bring up the biometrics correctly.

### Operating System

macOS

### Installation method

Mac App Store

### Build Version

版本 2024.8.2 Shell 31.3.1 Renderer 126.0.6478.185 Node 20.15.1 Architecture arm64

### Issue Tracking Info

- [ ] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by aimuz** [comment]
rel https://github.com/bitwarden/clients/pull/10374

It looks like it has something to do with this, but I can't verify it, only the app store version supports biometrics

**Comment by aimuz** [comment]
When will version 2024.9.0 be released in the app store?

**Comment by JstnMcBrd** [comment]
I am also experiencing this issue.

- Microsoft Edge extension, version 2024.9.1
- Windows 11

Biometric unlock suddenly stopped working - the Windows fingerprint scan popup does not appear. This is a new issue. Signing out/back in, restarting the computer, turning off/on biometric login - nothing has resolved it.

The Bitwarden desktop app is installed, up-to-date, unlocked, and ready to receive biometric login requests. Biometric login still works for the desktop app.

**Comment by aimuz** [comment]
The problem was solved when I updated to 2024.9.0，

macos system

**Comment by cbbit** [comment]
Hi there!

Thank you for your report, it seems like it is a duplicate of this one https://github.com/bitwarden/clients/issues/11168

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time in there - our engineering team will be happy to review these.

This issue will now be closed.

Thanks!
